src/AltinnCore/Designer/Repository/IDocumentDbRepository.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AltinnCore.Designer.Repository.Models;
+using AltinnCore.Designer.ViewModels.Request;
+using Microsoft.Azure.Documents;
+
+namespace AltinnCore.Designer.Repository
+{
+    /// <summary>
+    /// Interface
+    /// </summary>
+    public interface IDocumentDbRepository
```

Comment:
should interfaces also be renamed to match the {Model}Repository pattern ?

Reply:
Whoops, thought I renamed those aswell, doing it now.